cmd/network-slirp-binding/domain/domain.go

```diff
@@ -81,7 +81,23 @@ func (s SlirpNetworkConfigurator) Mutate(domainSpec *domainschema.DomainSpec) (*
 	if domainSpecCopy.QEMUCmd == nil {
 		domainSpecCopy.QEMUCmd = &domainschema.Commandline{}
 	}
-	domainSpecCopy.QEMUCmd.QEMUArg = append(domainSpecCopy.QEMUCmd.QEMUArg, slirpQemuCmdArgs...)
+
+	var currentCmdArgString string
+	for _, arg := range domainSpecCopy.QEMUCmd.QEMUArg {
+		currentCmdArgString += arg.Value
+	}
+
+	var generatedCmdArgString string
+	for _, arg := range slirpQemuCmdArgs {
+		generatedCmdArgString += arg.Value
+	}
+
+	if !strings.Contains(currentCmdArgString, generatedCmdArgString) {
+		domainSpecCopy.QEMUCmd.QEMUArg = append(domainSpecCopy.QEMUCmd.QEMUArg, slirpQemuCmdArgs...)
+		return domainSpecCopy, nil
+	}
+
+	log.Log.Warning("slirp qemu cmd args already exist, domain spec did not change")
```

Comment:
Why this scenario is considered a warning? Sounds reasonable to me with no need to log it.

Reply:
It can be considered as info, I added it to make debugging easier.

Reply:
It is not info, it is warning.
But I do not see the justification to log this all the time, it is not informative and except perhaps debugging it while developing, it would not help much in production.

Reply:
I removed the log message for now.